components/orderableTable/OrderableTable.scss

```diff
@@ -0,0 +1,18 @@
+.orderableHelper {
+    z-index: 666;
+    display: flex;
+    align-items: center;
+    color: black;
+    background: white;
+    border-top: 1px solid #dde6ec;
+    border-bottom: 1px solid #dde6ec;
+
+    td {
```

Comment:
As best practice we try to never do CSS nesting, could you add a class on this `td`?

Reply:
So I looked more into it, and the thing is that element with `orderableHelper` class is cloned node of element that user clicked on (see: https://github.com/clauderic/react-sortable-hoc/blob/master/src/SortableContainer/index.js#L299).

Because we're not rendering that element by ourselves, we can't add these classes w/o some hacks (e.g. `[...document.querySelector('.orderableHelper td')].forEach(node => node.classList.add('column'))`

Reply:
I see what you mean, then I'm OK to keep what you did.